### ISSUE TITLE
Skip to Main Content, Node 21

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#109]
Screengrab/videos (insert file names here)
2020 03 23, Skip to Main Content, Node 21
[2020 03 23, Skip to Main Content.zip](https://github.com/CityOfBoston/Iterators/files/4378160/2020.03.23.Skip.to.Main.Content.zip)
.mp4
2020 03 23, Skip to Main Content, Accessible site.mp4

Link
https://d8-stg.boston.gov

Node # (if known)
21

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > macOS > Safari

Trusted Tester Test
9-Repetitive Content

Brief Description
Repetitive content cannot be skipped using the keyboard

Steps to reproduce
1)	Refer to:  2020 03 23, Skip to Main Content, Node 21.mp4
2)	00:00, Start pressing the tab key to enter the main page
3)	00:09, The first element on the page is reached with the tab key.
a)	Bug:  This is the menu rather than a “Skip to Main Content” option which will allow a user with a screen reader to skip the repetitive menu and proceed directly to the main content.
4)	Refer to:  2020 03 23, Skip to Main Content, Accessible site.mp4
a)	00:00, Go to:  https://www.parrapark.com.au and start pressing the tab key to cycle onto the page
b)	00:05, The tab key results in reaching the first element on the page.  This element is “Skip to Main Content” which allows the user to press Enter and skip the menu in order to avoid screen readers processing this for every page.  Also, some non-screen reader users also use the keyboard for navigation and this avoids the repetitive content.

Note:  All major sections should also have a “skip” link to provide an option to keyboard users.  Examples of sections on this main page are:
-	Today’s status
-	Common Resources
-	Guides
-	Latest News
-	Featured Videos
-	311 Service Requests
-	Upcoming Events
-	Boston City Council
Otherwise, a user would have to press the tab key many times in order to go to the later sections on the page.

Desired Result
Repetitive content can be skipped when navigating with the keyboard.

Actual Result
Option is not available for skipping repetitive content.

**Comment by david-iterators** [comment]
Verified on https://d8-uat.boston.gov
